docs/how-to/integration_test_allure.md

```diff
@@ -0,0 +1,64 @@
+# Allure Report Integration
```

Comment:
Since this is now a how-to guide, I would rephrase the title in the style of `# How to...`. Maybe something like `# How to set up Allure Reports for integration tests` -- let me know if you have any thoughts or ideas :)

Reply:
addressed